ms-engine/src/main/java/net/trajano/ms/engine/jaxrs/CommonObjectMapperProvider.java

```diff
@@ -0,0 +1,32 @@
+package net.trajano.ms.engine.jaxrs;
+
+import javax.annotation.Priority;
+import javax.ws.rs.Priorities;
+import javax.ws.rs.ext.ContextResolver;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.fasterxml.jackson.module.jaxb.JaxbAnnotationModule'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)